web-components/src/api/robotoff.ts

```diff
@@ -0,0 +1,48 @@
+import { ROBOTOFF_API_URL, IS_DEVELOPMENT_MODE } from "../constants"
```

Comment:
I think robotoff api should be given as an attribute to the component.

We want to use it on staging for example.